packages/odyssey-react-mui/src/Field.tsx

```diff
@@ -26,11 +28,9 @@ import { useUniqueId } from "./useUniqueId";
 
 export const fieldTypeValues = ["single", "group"] as const;
 
+type FieldComponentPropsUsedByField = Omit<FieldComponentProps, "name">;
```

Comment:
I thought we were doing inline definitions and using `Pick` instead?

Reply:
This shouldn't be here. I'll make sure it gets removed